src/components/AdminDashboard/EmergentIssues.jsx

```diff
@@ -1,29 +1,29 @@
 import { React } from 'react';
-import { Text, Grid, GridItem } from '@chakra-ui/react';
+import { Text, Flex } from '@chakra-ui/react';
 import { PropTypes } from 'prop-types';
 
+import { useRowSelect } from 'react-table';
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'useRowSelect' is defined but never used.